packages/mobile/e2e/src/usecases/RestoreAccountOnboarding.js

```diff
@@ -1,16 +1,24 @@
-import { enterPinUi, sleep, waitForElementId } from '../utils/utils'
-import { SAMPLE_BACKUP_KEY } from '../utils/consts'
-
-const EXAMPLE_NAME = 'Test Name'
+import { enterPinUi, waitForElementId, sleep } from '../utils/utils'
```

Comment:
Is this a self-contained set of changes to specific test cases?  If so, it's a candidate for splitting into a separate PR to facilitate review.